einstein-ai/1.0/modules/ROOT/pages/einstein-ai-connector-reference.adoc

```diff
@@ -0,0 +1,660 @@
+| Locale a| String | Localization information, which can include the default locale, input locale(s), and expected output locale(s). |  en_US | 
+| Target Variable a| String | Name of the variable that stores the operation's output. |  | 
+| Target Value a| String |  Expression that evaluates the operation's output. The expression outcome is stored in the target variable. |  #[payload] | 
+| Error Mappings a| Array of <<ErrorMapping>> | Set of error mappings. |  | 
+| Reconnection Strategy a| * <<Reconnect>>
+* <<ReconnectForever>> |  Retry strategy in case of connectivity errors. |  | 
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+|Type |Binary
+| Attributes Type a| <<EinsteinResponseAttributes>>
+|===
+
+=== For Configurations
+
+* <<Config>> 
+
+=== Throws
+
+* MS-EINSTEIN:CHAT_FAILURE 
+* MS-EINSTEIN:CONNECTIVITY 
+* MS-EINSTEIN:RETRY_EXHAUSTED 
+
+
+[[ChatGenerateFromMessages]]
+== CHAT Generate From Messages
+`<ms-einstein:chat-generate-from-messages>`
+
+The *Chat generate from messages* operation is a prompt request operation with provided messages to the configured LLM. It uses a plain text prompt as input and responds with a plain text answer.
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| Messages a| String |  |  #[payload] | 
+| Output Mime Type a| String |  MIME type this operation outputs. |  | 
+| Output Encoding a| String |  Encoding of the payload this operation outputs. |  | 
+| Config Ref a| ConfigurationProvider |  Name of the configuration to use to execute this component. |  | x
+| Streaming Strategy a| * <<RepeatableInMemoryStream>>
+* <<RepeatableFileStoreStream>>
+* non-repeatable-stream | Configures how Mule processes streams. Repeatable streams are the default behavior. |  | 
+| Model API Name a| String | Name of the API model that interacts with the LLM. |  sfdc_ai__DefaultOpenAIGPT35Turbo | 
+| Probability a| Number | Probability of the model API staying accurate. |  0.8 | 
+| Locale a| String | Localization information, which can include the default locale, input locale(s), and expected output locale(s). |  en_US | 
+| Target Variable a| String |  Name of the variable that stores the operation's output. |  | 
+| Target Value a| String |  Expression that evaluates the operation's output. The expression outcome is stored in the target variable. |  #[payload] | 
+| Error Mappings a| Array of <<ErrorMapping>> |  Set of error mappings. |  | 
+| Reconnection Strategy a| * <<Reconnect>>
+* <<ReconnectForever>> |  Retry strategy in case of connectivity errors. |  | 
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+|Type |Binary
+|===
+
+=== For Configurations
+
+* <<Config>> 
+
+=== Throws
+
+* MS-EINSTEIN:CHAT_FAILURE 
+* MS-EINSTEIN:CONNECTIVITY 
+* MS-EINSTEIN:RETRY_EXHAUSTED 
+
+
+[[EmbeddingAdhocFileQuery]]
+== EMBEDDING Adhoc File Query
```

Comment:
I thought there were three Embedding operations as per the Configuring Embedding operations doc?

Reply:
Good catch. Looks like one is missing. We'll probably publish without it and add it later.

Reply:
It's there now.